node/sc/vt_recovery.go

```diff
@@ -205,12 +205,16 @@ func updateRecoveryHintFromTo(prevHint *valueTransferHint, from, to *BridgeInfo)
 	if err != nil {
 		return nil, err
 	}
+	from.SetRequestNonce(requestNonce)
```

Comment:
`vtrecovery` is optional. Should we update the nonce-related information here?

Reply:
You are right. First time I tried to make another routine to retrieve and update. 
But it was a duplicated with this code.
So I changed it.

I think VT recovery is almost default feature. 
So It's time to set vtrecovery as an the default option. :) 
What do you think?

Reply:
I agree with you. `vtrecovery` is actually not an option anymore. Let's make it a default.